### DISCUSSION TITLE
Combobox Input doesn't respond until 2nd character is typed

### DISCUSSION THREAD

**Discussion by magnum6actual** [tailwindlabs/headlessui#1241] [Help]
I've got a pretty basic implementation of the combobox component in Vue.  I've noticed that when I click on the combobox input, even though it has the focus and is showing the blinking cursor, the first character typed is always ignored and the input doesn't change until a second character is typed.

Not sure what's going on. Is this designed behavior for some reason?

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

It's definitely not designed this way and can't reproduce it here either: https://headlessui-9nyq41ami-tailwindlabs.vercel.app/combobox/combobox-with-pure-tailwind 
Can you try and reproduce it and create a GitHub repo / CodeSandbox and share it?
It might also help to install the latest insiders version to see if that fixes things for you.
